packages/frontend-2/lib/user/graphql/queries.ts

```diff
@@ -21,3 +21,13 @@ export const userEmailsQuery = graphql(`
     }
   }
 `)
+
+export const userActiveWorkspaceQuery = graphql(`
```

Comment:
also - what's this for? im asking cause i saw multiple queries already asking for similar data

Reply:
In some places we need just the active workspace slug in order to fetch additional workspace info using a workspaceBySlug query for example, I've renamed it to useActiveWorkspaceSlug to make it a bit clearer that this is meant to just get the slug, not the full activeWorkspace